ui/pages/Project/components/Matchmaker.jsx

```diff
@@ -160,28 +161,31 @@ const CONTACT_FIELDS = [
   { name: 'body', component: BaseSemanticInput, inputType: 'TextArea', rows: 12 },
 ]
 
-const BaseContactHostButton = React.memo(({ defaultContactEmail, onSubmit }) => (
+const BaseContactHostButton = React.memo(({ defaultContactEmail, onSubmit, canSend }) => (
   <UpdateButton
-    onSubmit={onSubmit}
+    onSubmit={canSend && onSubmit}
     initialValues={defaultContactEmail}
     formFields={CONTACT_FIELDS}
-    modalTitle={`Send Contact Email for Patient ${defaultContactEmail.patientId}`}
+    modalTitle={`${canSend ? 'Send' : 'Draft'} Contact Email for Patient ${defaultContactEmail.patientId}`}
     modalId={`contactEmail-${defaultContactEmail.patientId}`}
     buttonText="Contact Host"
     editIconName="mail"
     showErrorPanel
     submitButtonText="Send"
     buttonFloated="right"
+    submitOnChange={!canSend}
```

Comment:
hides the submit button when true

Reply:
This is kinda sneaky, can you add this as a comment in the formwrapper component above the submitOnChange prop?